FiveCalls/NotificationsService/NotificationService.swift

```diff
@@ -0,0 +1,36 @@
+//
```

Comment:
we have 2 of these?

Reply:
Weird. Seems like the one signal docs suggest modifying the one in the notification extension, I'll keep that one.